tools/IoTEdgeDevOps/DevOpsLib/AzureLogAnalytics.cs

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace DevOpsLib
+{
+    using System;
+    using System.Net.Http;
+    using System.Threading.Tasks;
+
+    public sealed class AzureLogAnalytics
+    {
+        static readonly AzureLogAnalytics instance = new AzureLogAnalytics();
```

Comment:
why do you want to make this as singleton?

Reply:
I could.  Previously it was couple with ADD which SHOULD really be a singleton.
I can make change to this so that it is not a singleton.